app/services/support_interface/tad_provider_stats_export.rb

```diff
@@ -1,42 +1,47 @@
 module SupportInterface
   class TADProviderStatsExport
-    def call
-      Course
-        .includes(:provider)
-        .map { |c| course_to_row(c) }
-        .sort_by { |r| r[:provider_code] }
-    end
+    def data_for_export
+      data_for_export = courses.map do |course|
+        choice_statuses = choice_statuses(course)
 
-    alias_method :data_for_export, :call
+        {
+          provider_id: nil,
```

Comment:
I assumed these `nil` columns are here for a reason. But if not can we remove them? @duncanjbrown

Reply:
Yea i don't have any context on this, It does seem strange, but maybe they match and populate these columns on their end with their id's?